pkg/apis/policy/validation/validation.go

```diff
@@ -50,8 +50,20 @@ func ValidatePodDisruptionBudgetUpdate(pdb, oldPdb *policy.PodDisruptionBudget)
 func ValidatePodDisruptionBudgetSpec(spec policy.PodDisruptionBudgetSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	allErrs = append(allErrs, extensionsvalidation.ValidatePositiveIntOrPercent(spec.MinAvailable, fldPath.Child("minAvailable"))...)
-	allErrs = append(allErrs, extensionsvalidation.IsNotMoreThan100Percent(spec.MinAvailable, fldPath.Child("minAvailable"))...)
+	if spec.MinAvailable != nil && spec.MaxUnavailable != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath, spec, "MinAvailable and MaxUnavailable cannot be both set"))
+	}
+
```

Comment:
I think we should also validate that MaxUnavailable != 0 when it's set so we can allow node drains to proceed.  cc: @davidopp @smarterclayton  Unfortunately we cannot validate MinAvailable likewise .. @deads2k is there something that we need to know regarding cross-object validation or that is not a thing in the api-machinery plans?